<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCategoryDialog</class>
 <widget class="QDialog" name="NewCategoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add category</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="categoryLabel">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="categoryEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="placeLabel">
          <property name="text">
           <string>Place:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="placeSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
